File: ElasticErrorRates.API/SignalR/SignalRMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ElasticErrorRates.API.SignalR
{
    public class SignalRMessage
    {
        public string Type { get; set; }
        public string Payload { get; set; }
    }
}

File: ElasticErrorRates.UI/src/app/_shared/signalR/signalR.message.ts
export interface ISignalRMessage {
    type: string;
    payload: string;
}

export class SignalRMessage implements ISignalRMessage {
    
    type: string;
    payload: string;
    
    constructor(type: string, payload: string) {
        this.type = type;
        this.payload = payload;
    }

    
}
